tests/greenheart/test_hydrogen/input_files/plant/GS_greenheart_config.yaml

```diff
@@ -0,0 +1,194 @@
+site:
+  mean_windspeed: False 
+project_parameters:
+  grid_connection: False # option, can be turned on or off
+  grid_to_electrolyzer_input: 'hydrogen' #'power' or 'hydrogen'
```

Comment:
Can you explain the reasoning behind the option names here? I don't understand.

Reply:
in line 5

Reply:
changed "grid_to_electrolyzer_input" under "project_parameters" to "grid_input_signal" under "electrolyzer"